code/modules/cargo/packs.dm

```diff
@@ -2167,14 +2167,14 @@
 
 /datum/supply_pack/organic/cream_piee
 	name = "High-yield Clown-grade Cream Pie Crate"
-	desc = "Designed by Aussec's Advanced Warfare Research Division, these high-yield, Clown-grade cream pies are powered by a synergy of performance and efficiency. Guaranteed to provide maximum results."
+	desc = "Designed by Aussec's Advanced Warfare Research Division, these high-yield, Clown-grade cream pies are powered by a synergy of performance and efficiency. Guaranteed to provide maximum results. Requires Clown to open."
```

Comment:
Weird grammar
```suggestion
	desc = "Designed by Aussec's Advanced Warfare Research Division, these high-yield, Clown-grade cream pies are powered by a synergy of performance and efficiency. Guaranteed to provide maximum results. Requires Clown Access to open."
```

Reply:
I just put Clown because there's no clown specific access, and it only checks for if the person is a clown instead of any access + it looks funnier

Reply:
I agree that it is funnier this way. It is true that you need a clown specifically, not any traditional "Access"